### DISCUSSION TITLE
remak-iframes

### DISCUSSION THREAD

**Discussion by Christonn93** [remarkjs/remark#1290] [Q&A] [answered]
Just wondering if this feature will be working in the near feature?

**Comment by wooorm** [comment]
What feature? What doesn’t work? Can you please provide more info?

**Comment by ChristianMurphy** [comment]
Welcome @Christonn93! 👋 
`remark-iframes` is maintained by https://github.com/zestedesavoir

They have a preview version which works with the latest version of `remark` feel free to test it out and contribute back if you see any issues https://github.com/zestedesavoir/zmarkdown/issues/416#issuecomment-1419768957

**Comment by Christonn93** [comment]
@ChristianMurphy & @wooorm 

I'm talking about [this](https://github.com/zestedesavoir/zmarkdown/tree/HEAD/packages/remark-iframes#readme) 
I cant get it to work. Might be me who dont know exactly how to use it, I will give it another go. 

What im trying to do is to have a way to just add the link to markdown and have it change into a iframe. 
I would like to have it in this way `@[Video-title](Video.link.com)`, but that is a syntax that create 2 different nodes. 
The `@` is type text, and `[]()` is type link. 

But I will have one more go at the iframe that i linked to, maybe i need to understand it better. 

Thanks anyway :) 
 

PS:

The project im working on is a svelte project. And its using MDsvelX.

**Reply by wooorm** [reply, marked as answer]
If you are looking for a generic component format, you are looking for `remark-directive`.
But you still need to handle those components.

Alternatively, use links: `[Title](https://asdasdasd "is:video")`, then with a transform plugin, you can look for links that have a `title` of `is:video`, and transform them. This one is pretty similar to your goal: https://github.com/rehypejs/rehype-picture.

MDsvelX used to be very outdated. Not sure if that’s still the case. You can also use MDX itself with Svelte. MDX already has a component syntax: `<Video id="asdasdasd" />`
